model/src/pyrenew/latent/hospitaladmissions.py

```diff
@@ -63,73 +63,76 @@ class HospitalAdmissions(RandomVariable):
 
     def __init__(
         self,
-        infection_to_admission_interval: RandomVariable,
-        infect_hosp_rate_dist: RandomVariable,
-        admissions_predicted_varname: str = "predicted_admissions",
-        weekday_effect_dist: Optional[RandomVariable] = None,
-        hosp_report_prob_dist: Optional[RandomVariable] = None,
+        infection_to_admission_interval_rv: RandomVariable,
+        infect_hosp_rate_rv: RandomVariable,
+        observed_hosp_admissions_varname: str = "observed_hosp_admissions",
+        weekday_effect_rv: Optional[RandomVariable] = None,
+        hosp_report_prob_rv: Optional[RandomVariable] = None,
```

Comment:
This is a minor nuisance grandfathered in from before we decided to cease using the semi-outdated `from typing import Optional` and instead use `|`. 

```suggestion
        weekday_effect_rv: RandomVariable | None = None,
        hosp_report_prob_rv: RandomVariable | None = None,
```